org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/EventHub.java

```diff
@@ -1,4 +1,4 @@
-package org.eclipse.jdt.ls.debug.internal;
+package org.eclipse.jdt.ls.debug;
```

Comment:
add a new interface IEventHub and keep EventHub implementation in internal package?